09/README.md

```diff
@@ -101,7 +99,7 @@ Pay close attention to the function ```rotateTilePattern()```, which subdivides
 
 * Comment, uncomment and duplicate lines 69 to 72 to compose new designs.
 
-* Change the black and white triangle for another element like: half circles, rotated squares or lines.
+* Change the black and white triangle for another element: like half circles, rotated squares or lines.
```

Comment:
Better would be to remove the colon entirely.

Reply:
Mmmmm let me consult Jen on this one